### ISSUE TITLE
Custom user permissions error

### ISSUE THREAD

**Issue by milesjbland** [ryssbowh/craft-activity#4]
Hi there,

I've added some custom user permissions via a module:

```
Event::on(
    UserPermissions::class,
    UserPermissions::EVENT_REGISTER_PERMISSIONS,
    function(RegisterUserPermissionsEvent $event) {
        $event->permissions[\Craft::t('plugin-handle', 'Exports')] = [
            'viewUserExport' => ['label' => \Craft::t('plugin-handle', 'View User Export')],
            'viewResourceExport' => ['label' => \Craft::t('plugin-handle', 'View Resource Export')],
        ];
    }
);
```

When I update the custom permissions on a user and save, I get the following error:

```
yii\base\ErrorException: Undefined index: permissions in /vendor/ryssbowh/craft-activity/src/models/fieldHandlers/projectConfig/Permissions.php:120
Stack trace:
#0 /vendor/craftcms/cms/src/web/ErrorHandler.php(87): yii\base\ErrorHandler->handleError(8, 'Undefined index...', '...', 120)
#1 /vendor/ryssbowh/craft-activity/src/models/fieldHandlers/projectConfig/Permissions.php(120): craft\web\ErrorHandler->handleError(8, 'Undefined index...', '...', 120, Array)
#2 /vendor/ryssbowh/craft-activity/src/models/fieldHandlers/projectConfig/Permissions.php(102): Ryssbowh\Activity\models\fieldHandlers\projectConfig\Permissions->findPermission('viewuserexport')
#3 /vendor/ryssbowh/craft-activity/src/models/fieldHandlers/projectConfig/Permissions.php(87): Ryssbowh\Activity\models\fieldHandlers\projectConfig\Permissions->labelledPermissions(Array)
#4 /vendor/ryssbowh/craft-activity/src/models/fieldHandlers/projectConfig/Permissions.php(47): Ryssbowh\Activity\models\fieldHandlers\projectConfig\Permissions->buildDirty(Array, Array)
#5 /vendor/ryssbowh/craft-activity/src/recorders/Users.php(320): Ryssbowh\Activity\models\fieldHandlers\projectConfig\Permissions->getDirty(Object(Ryssbowh\Activity\models\fieldHandlers\projectConfig\Permissions))
#6 /vendor/ryssbowh/craft-activity/src/recorders/Users.php(345): Ryssbowh\Activity\recorders\Users->saveNewPermissions(Object(craft\web\Application))
#7 [internal function]: Ryssbowh\Activity\recorders\Users->Ryssbowh\Activity\recorders\{closure}(Object(yii\base\Event))
#8 /vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(yii\base\Event))
#9 /vendor/yiisoft/yii2/base/Component.php(642): yii\base\Event::trigger('yii\\base\\Applic...', 'afterRequest', Object(yii\base\Event))
#10 /vendor/yiisoft/yii2/base/Application.php(387): yii\base\Component->trigger('afterRequest')
#11 /web/index.php(22): yii\base\Application->run()
#12 /.composer/vendor/laravel/valet/server.php(234): require('...')
#13 {main}
```

For now, I can refresh the page and it seems to update fine. But, it would be good if the plugin was compatible with custom permissions.

Craft CMS: **3.7.61**
User Activity: **1.2.3**

**Comment by ryssbowh** [comment]
Hi,

Thanks for reporting, this is fixed in 1.2.4

**Comment by milesjbland** [comment]
Just updated and it's all working as expected now. Thanks for sorting so quickly!
